apps/codeimage/src/pages/Dashboard/components/ProjectList/ProjectErrorListFallback.tsx

```diff
@@ -0,0 +1,34 @@
+import {Box, Button, Text} from '@codeimage/ui';
+import {VoidProps} from 'solid-js';
+import {ExclamationAltIcon} from '../../../../components/Icons/Exclamation';
+import * as styles from './ProjectList.css';
+
+interface Props {
+  error?: unknown;
+
+  onReload?(): void;
+}
+
+export function ProjectErrorListFallback(props: VoidProps<Props>) {
+  // TODO: add i18n
+
+  return (
+    <div class={styles.fallbackContainer}>
+      <ExclamationAltIcon size={'3x'} />
+      <Text size={'2xl'} class={styles.fallbackTextTitle}>
```

Comment:
Al piu presto va refactorato il componente `Text` in modo da poter utilizzare i colori base o il colore `description` del tema